### ISSUE TITLE
[Keyboard Navigation : User unable to access 'Insert link' alert dialog which is displayed by invoking 'Insert Link' control present in 'Rich text editor toolbar.

### ISSUE THREAD

**Issue by IyyappanRam2** [ckeditor/ckeditor5#10999]
## Type of report

Keyboard Navigation on CK  Editor Rich Text box

## reproduction steps 

Using Keyboard Controls User unable to access 'Insert link' alert dialog which is displayed by invoking 'Insert Link' control present in 'Rich text editor toolbar.

### Expected result

User should be able to access 'Insert link' alert dialog which is displayed by invoking 'Insert Link' control present in 'Rich text editor toolbar.
Ex: Once the dialog is displayed, keyboard focus should land on the edit field present in the dialog.

### Actual result
User unable to access 'Insert link' alert dialog which is displayed by invoking 'Insert Link' control present in 'Rich text editor toolbar.
Ex: 
1.Once the dialog is displayed, keyboard focus is landing on close(X) button in the Given Feedback dialog.
2.Keyboard focus is not at all moving to the displayed alert dialog

## Other details

Browser: MIcrosoft Edge (Chromium) & Google Chrome
OS: Windows 10
CKEditor version: @ckeditor/ckeditor5-react : 3.0.0
Installed CKEditor plugins: @ckeditor/ckeditor5-build-decoupled-document : 25.0.0

**Comment by IyyappanRam2** [comment]
@KarolDawidziuk could you provide me the respective link for the above update

**Comment by IyyappanRam2** [comment]
@KarolDawidziuk Please re-open this ticket
